packages/bierzo-wallet/package.json

```diff
@@ -40,23 +40,20 @@
-    "use-config-staging": "mkdir -p ./build/static/config && cp ./config/staging.json ./build/static/config/conf.json",
-    "use-config-production": "mkdir -p ./build/static/config && cp ./config/production.json ./build/static/config/conf.json",
+    "override-config-development": "mkdir -p ./build/static/config && cp ./src/config/development.json ./build/static/config/conf.json",
+    "override-config-staging": "mkdir -p ./build/static/config && cp ./src/config/staging.json ./build/static/config/conf.json",
+    "override-config-production": "mkdir -p ./build/static/config && cp ./src/config/production.json ./build/static/config/conf.json",
     "lint": "tsc --noEmit && eslint -c .eslintrc.js --max-warnings 0 'src/**/*.ts{,x}'",
     "format": "prettier --write --loglevel warn './src/**/*.{ts,tsx,json,md,css}'",
-    "start-development": "yarn use-config-development && react-scripts start",
-    "start-staging": "yarn use-config-staging && react-scripts start",
-    "start-production": "yarn use-config-production && react-scripts start",
-    "start": "yarn start-development",
-    "build-development": "yarn use-config-development && react-scripts build",
-    "build-staging": "yarn use-config-staging && react-scripts build",
-    "build-production": "yarn use-config-production && react-scripts build",
+    "start": "react-scripts start",
+    "build-development": "yarn react-scripts build && yarn override-config-development",
+    "build-staging": "react-scripts build && yarn override-config-staging",
+    "build-production": "react-scripts build && yarn override-config-production",
     "build": "yarn build-development",
-    "test-dom": "yarn use-config-development && CI=true react-scripts test '\\.dom\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom",
-    "test-unit": "yarn use-config-development && CI=true react-scripts test '\\.unit\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom --detectOpenHandles",
-    "test-e2e": "yarn use-config-development && yarn use-extension-config-development && CI=true react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom --runInBand",
-    "test-single": "yarn use-config-development && yarn use-extension-config-development && CI=true react-scripts test --env=jsdom --detectOpenHandles",
+    "test-dom": "yarn override-config-development && CI=true react-scripts test '\\.dom\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom",
+    "test-unit": "yarn override-config-development && CI=true react-scripts test '\\.unit\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom --detectOpenHandles",
+    "test-e2e": "yarn override-config-development && yarn use-extension-config-development && CI=true react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom --runInBand",
+    "test-single": "yarn override-config-development && yarn use-extension-config-development && CI=true react-scripts test --env=jsdom --detectOpenHandles",
```

Comment:
We do not need to override configs because of:
```
if (process.env.NODE_ENV === "test") {
    return (window as WindowWithConfig).developmentConfig;
  }
```

Reply:
Exactly. The config file in build/ was not used anyways. However, we need to keep it in the e2e tests that start a webserver serving ./build data

Reply:
You can leave it for e2e tests only.

Reply:
Updated in https://github.com/iov-one/ponferrada/pull/482/commits/c649e4e6ecc5b8f0d3f025cfc03f298b246c1d07